src/ModelHelper.php

```diff
@@ -43,6 +43,17 @@ public function indexOnly(Model $model, $index_name)
         return !method_exists($model, 'indexOnly') || $model->indexOnly($index_name);
     }
 
+    public function wouldBeIndexed(Model $model, $index_name)
+    {
+        if (! method_exists($model, 'indexOnly')) {
+            return false;
+        }
+
+        $model->setRawAttributes($model->getOriginal());
```

Comment:
I think even if it's not returned, it will modify it outside because object are passed by reference, and not copy.
I think we need to clone the model object